frontend/src/modules/displays/ToggleValue.tsx

```diff
@@ -45,6 +49,16 @@ const StyledToggleButton = withStyles(({ palette }) => ({
   },
 }))(ToggleButton);
 
+/**
+ * @typedef Props
+ *
+ * Props Interface for the ToggleValue component
+ *
+ * @prop {string} toggleBetween.label Toggle Input label
+ * @prop {number} toggleBetween.value Toggle Input value
+ * @prop {number} selected Selected toggle value
```

Comment:
What's the relationship between `toggleBetween.value` and `selected`, since they're both toggle input values?

Reply:
`toggleBetween` provides input array of object to toggle between & `selected` is current selected value among them